CHANGES.md

```diff
@@ -1,5 +1,18 @@
 # Release Notes
 
+## 2.20.4
+
+This release fixes a ug where `--pip-log <path>`, used multiple times
```

Comment:
```suggestion
This release fixes a bug where `--pip-log <path>`, used multiple times
```

Reply:
I'm having a hell of a time getting CI green today. I may skip this until post merge and just manually edit the Releases / Announcements pages post-release. We'll see.

Reply:
I don't think it matters at all. Fine to fix later or never.

Reply:
Fixed. I had to add 2 new fixes to the change log, so there was a good moment to do this.